File: python/1.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
# @Author: ben_cripps
# @Date:   2014-12-06 14:39:05
# @Last Modified by:   ben_cripps
# @Last Modified time: 2014-12-06 14:45:41

print sum([i for i in xrange(1001) if i % 3 == 0 or i % 5 == 0])

File: js/1.js
/* 
* @Author: ben_cripps
* @Date:   2014-12-05 18:58:56
* @Last Modified by:   ben_cripps
* @Last Modified time: 2016-08-28 09:51:07
*/

'use strict';

for (var sum = 0, i = 0; i < 1000; i++) {
    if (i % 3 === 0 || i % 5 === 0) {
        sum += i;
    }
}

console.log(sum);